### PR TITLE
Update/licenses refinement

### PR DESCRIPTION
**Pull request by leogr**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

/kind cleanup

> /kind design

/kind documentation

> /kind failing-test

> /kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area build

> /area driver-kmod

> /area driver-ebpf

> /area libscap

> /area libsinsp

> /area tests

**What this PR does / why we need it**:

Small adjustments to licenses.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by leogr** [comment]
Sorry for spamming, I'm using this PR to test the CI job.

/test build-libs-bundled-deps

File: NOTICES [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+
```
Comment (leodido) [inline NOTICES:2]:
```suggestion
```

Why these empty lines?

**Review by leodido — COMMENTED** [review]
SGTM

File: NOTICES (hunk 1 above)
Comment (leogr) [reply NOTICES:2]: Leftover, I'm fixing

**Review by leodido — APPROVED** [review]

**Review by fntlnz — APPROVED** [review]

**Pull request merged**